app/views/groups/members/index.html.erb

```diff
@@ -16,6 +16,25 @@
       <div class="shadow">
         <%= turbo_frame_tag("member-update-alert") %>
         <table class="min-w-full table-fixed dark:divide-gray-600">
+          <thead>
```

Comment:
**suggestion (blocking):** Not a fan of the table header design.  I have been mostly following what Flowbit does for their stuff.   This is their table header:

![image](https://github.com/phac-nml/irida-next/assets/11295750/59fd5f7b-0331-4199-926c-3c54ad21d618)

@ericenns made a modification of that for his "Active Personal Access Tokens" table that you could take a look at.  Some point we will need to make a component for that.

Reply:
I updated the table header design to follow how it looks on the personal access tokens page for now. Once we create a component for the table header we can update these pages. Updated in d212883497cdc5da02bc5ca354c0cea05d7f9afd

Reply:
Looks great, thanks.